README.md

```diff
@@ -187,6 +187,20 @@ GitHub github = Feign.builder()
                      .target(GitHub.class, "https://api.github.com");
 ```
 
+If you need to pre-process the response before give it to the Decoder, you can use the [InterceptorDecoder](/feign/codec/InterceptorDecoder.java).
+
+```java
+GitHub github = Feign.builder()
+                     .decoder(
+                         new InterceptorDecoder(
```

Comment:
thanks for be patient!
I'm not sure that I understand what do you want to say with "static package protected member of feign"...
make the `ResponseMappingDecoder` an inner class insied the `feign.Feign` abstract class?